Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/properties/PropertyAccessor.java

```diff
@@ -40,8 +40,8 @@
  */
 public class PropertyAccessor implements IPropertyAcessor {
 
```

Comment:
Why was this class edited?  Seems like the only thing that was changed is final variables which I'm not sure is needed for this PR.